plugin/src/main/java/org/autorefactor/refactoring/Refactorings.java

```diff
@@ -413,4 +413,13 @@ public BadLocationException call() throws Exception {
             throw ex;
         }
     }
+
+    /**
+     * Gets the ASTRewrite rewrite object.
+     *
+     * @return the ASTRewrite rewrite
+     */
+    public ASTRewrite getRewrite() {

```

Comment:
No, I think you used "Encapsulate Field" refactoring because this is exactly the result you would get in this case.